sycl/test-e2e/lit.cfg.py

```diff
@@ -815,6 +815,6 @@ def open_check_file(file_name):
 try:
     import psutil
 
-    lit_config.maxIndividualTestTime = 600
```

Comment:
I'm against this change. Our tests are slow as it is, Allowing them to take 2x time is a bad idea and tests should still be split further if they reach timeout

Reply:
oh, if this PR is impacting all tests, then change in this file is not good

Reply:
Got it. I've reverted the timeout change.